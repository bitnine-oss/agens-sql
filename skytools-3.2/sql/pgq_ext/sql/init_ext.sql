
\set ECHO off
\i structure/install.sql
\set ECHO all
create extension pgq_ext from 'unpackaged';
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_ext';
drop extension pgq_ext;

create extension pgq_ext;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_ext';

