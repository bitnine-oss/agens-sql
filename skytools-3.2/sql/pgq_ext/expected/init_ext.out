\set ECHO off
 upgrade_schema 
----------------
              4
(1 row)

create extension pgq_ext from 'unpackaged';
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_ext';
 dumpable 
----------
        4
(1 row)

drop extension pgq_ext;
create extension pgq_ext;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_ext';
 dumpable 
----------
        4
(1 row)

